src/locales/en.json

```diff
@@ -377,6 +377,7 @@
   "benefit_letters": "Benefits Letter",
   "government_letter": "Government Letter",
   "council_tax": "Council Tax Letter",
+  "loading": "Loading...",
```

Comment:
Minor: not sure if I would like an ellipsis to be read out

Reply:
It doesn't seem to make any difference in the way the screen reader reads the `...`. I can get rid of them